assembl/static2/js/app/pages/debate.jsx

```diff
@@ -30,11 +34,41 @@ class Debate extends React.Component {
   displayThumbnails() {
     this.setState({ isThumbnailsHidden: !this.state.isThumbnailsHidden });
   }
+  redirectToV1Threads() {
```

Comment:
@ayyazdaniaryan I prefer in `main.jsx` than in `utils` because it's a UI behavior. Do you have a better argument?